script/screenshots.js

```diff
@@ -0,0 +1,142 @@
+// This script assumes the following:
+//   1. You've installed wd with `npm install wd'.
+//   2. You've set the environment variables $SAUCE_USERNAME and $SAUCE_ACCESS_KEY.
+//   3. If the environment variable $CIRCLE_ARTIFACTS is not set imanges will be saved in /tmp
```

Comment:
could be nice to describe the directory structure this produces, since there's other ways it could've gone (e.g. one folder has  all the final images, another has the folders of pieces), and it helps for understanding the 3 `dir` variables